composer.json

```diff
@@ -41,12 +41,15 @@
         }
     },
     "autoload-dev": {
+        "classmap": ["tests/Application/src/Kernel.php"],
         "psr-4": {
-            "Tests\\MonsieurBiz\\SyliusAlertMessagePlugin\\": "tests/"
-        },
-        "classmap": ["tests/Application/Kernel.php"]
+            "App\\": "tests/Application/src/"
```

Comment:
yes, we should create an issue for that, because we have many plugins with the autoload definition ...
But, it's an `autoload-dev` instruction. So, it's not used during when installing the plugin i think ;)